### ISSUE TITLE
ADD icons

### ISSUE THREAD

**Issue by APakrohk** [gusbemacbe/suru-plus#62]
Perspolise DM 
https://persepolisdm.github.io/
Godot Game Engine
https://godotengine.org/
RamBox 
https://rambox.pro/#home
itch Game Store
https://itch.io
OpenSpades
https://github.com/yvt/openspades
############################################################
plz add :1st_place_medal:  :+1:

**Comment by gusbemacbe** [comment]
Hi @APakrohk 

When I will be done wit my thesis which must be delivered on the day 7, I will focus on the icons.

**Comment by BlueManCZ** [comment]
Hello,
I hope your thesis went well.

I have some other programs without icons:
[DrRacket](https://racket-lang.org/), [pgAdmin 4](https://www.pgadmin.org/), [Unreal Engine 4](https://github.com/EpicGames/UnrealEngine) and [Battle.net](https://www.blizzard.com/en-us/apps/battle.net/desktop) (on Wine).

This icon-set is great. I cheer you!

**Comment by gusbemacbe** [comment]
Sorry for my absence, I was very busy with my college, I had to finish my heavier college jobs which took this whole month, but I am finished with the last two jobs.

I will return to the day 8 December to review the icons themes after finishing my last college jobs, introducing my thesis and getting graduated.

**Comment by mbacou** [comment]
Also would like to suggest:
- RStudio
https://www.rstudio.com/
- Stata
https://www.stata.com/
- Python Jupyter notebooks
http://jupyter.org/
- GRASS GIS
https://grass.osgeo.org/
- Zoom
https://zoom.us/

Excellent icon theme. Thank you!

**Comment by gusbemacbe** [comment]
Hi @APakrohk, @BlueManCZ, @mbacou and @jonathonf !

All the icons have been added to the branch [unstable-2](/gusbemacbe/suru-plus/tree/unstable-2), but the small icons, not yet.

I have increased from 2260 to 4630 icons in the branch [unstable-2](/gusbemacbe/suru-plus/tree/unstable-2). This unstable version of icons theme will support officially the Steam games and other games icons. Officially supported all Flatpak icons.
